<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>572</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ImageResizerCUDA</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Image:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="pathBox"/>
      </item>
      <item>
       <widget class="QPushButton" name="browseBtn">
        <property name="text">
         <string>Browse...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="loadBtn">
        <property name="text">
         <string>Load</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="fitToWindowBtn">
        <property name="text">
         <string>Fit to window</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="resetZoomBtn">
        <property name="text">
         <string>Reset zoom</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Resize image to:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="widthBtn">
        <property name="text">
         <string>Width:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="widthSpinBox">
        <property name="maximum">
         <number>999999</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="heightBtn">
        <property name="text">
         <string>Height:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="heightSpinBox">
        <property name="maximum">
         <number>999999</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="ratioBtn">
        <property name="text">
         <string>Ratio:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="ratioSpinBox">
        <property name="minimum">
         <double>0.010000000000000</double>
        </property>
        <property name="maximum">
         <double>999999.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="resizeBtn">
        <property name="text">
         <string>Resize</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="resetBtn">
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="saveAsBtn">
        <property name="text">
         <string>Save as...</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Interpolation method to use:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="methodBox">
        <item>
         <property name="text">
          <string>Nearest neighbor (CUDA)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Bilinear (CUDA)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Nearest neighbor (CPU)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Bilinear (CPU)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Bicubic (CPU)</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="GraphicsViewEx" name="graphicsView"/>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GraphicsViewEx</class>
   <extends>QGraphicsView</extends>
   <header>graphicsviewex.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
